Sources/Confidence/Confidence.swift

```diff
@@ -1,23 +1,40 @@
 import Foundation
 
 public class Confidence: ConfidenceEventSender {
-    public var context: ConfidenceStruct
+    private let parent: ConfidenceEventSender?
+    private var localContext: ConfidenceStruct
+
+    public var context: ConfidenceStruct {
+        get {
+            var mergedContext = parent?.context ?? [:]
+            localContext.forEach { entry in
+                mergedContext.updateValue(entry.value, forKey: entry.key)
+            }
+            return mergedContext
+        }
+        set {
+            self.localContext = newValue
+        }
+    }
     public let clientSecret: String
     public var timeout: TimeInterval
     public var region: ConfidenceRegion
     public var initializationStrategy: InitializationStrategy
 
-    init(
+    required public init(
```

Comment:
With this necessary change, I am now thinking we could drop the Builder pattern and just use constructors with optional arguments

Reply:
This would anyway be part of a separate PR